src/utils/route3.util.ts

```diff
@@ -93,12 +93,14 @@ export const getRoute3TokenSymbol = (token: TokenInterface) => {
   return token.symbol;
 };
 
+export const getRoute3TokenSlug = ({ contract, tokenId }: Route3Token) => toTokenSlug(contract ?? '', tokenId ?? 0);
```

Comment:
The token for 3route Tezos token is exactly `'tez'` because its `contract` property is `null`, so [toTokenSlug](https://github.com/madfish-solutions/templewallet-mobile/blob/development/src/token/utils/token.utils.ts#L11) function receives an empty string and returns the expected value